.github/workflows/test.yml

```diff
@@ -60,17 +62,41 @@ jobs:
 
       - run: conda info
 
+      - name: Install Julia on Windows
+        if: matrix.os == 'windows-latest'
+        run:
+          pip install julia
+
+      - name: Install Julia on Ubuntu/OSX
+        if: matrix.os != 'windows-latest'
+        run:
+          conda install -yq julia[version">=1.10.0"]
+
+      - name: Install benchopt and its dependencies Windows
+        if: matrix.os == 'windows-latest'
+        run: |
+          call conda info
+          call conda install -yq pip
+          call pip install -e .[test]
+          call echo using conda
```

Comment:
I would not duplicate this step but create a variable `CONDA_CMD` in the `yaml` env section that is `conda` on linux and `call conda` on windows.
You can probably reuse `conda_cmd` from the matrix.

Reply:
@tomMoral the last command being written in bash script, I've tried to [condition](https://github.com/benchopt/benchopt/pull/717/commits/cf212a9300cf04c7f76aa509994b2eae617d6837) on the running OS before executing it, but no luck yet. Do you have any solution to that?

Also as you'll probably notice, I've moved the comment outside the run block because `cmd` seems to dislike `#`s.